### PR TITLE
get_resource default value

### LINKED ISSUE #47: get_resource() default value
```twig
{{ get_resource('contact_page.zoom', 15) }}
```
second parameter that will be returned if value not exist (`=== null`)

### PR DESCRIPTION
**Pull request by JarekW**
resolves #47

### CODE REVIEW

File: Twig/Extension/ResourceRepository.php [hunk 1]
```diff
@@ -59,8 +59,14 @@ public function hasResource($key)
...
      * @return mixed|null
      */
-    public function getResource($key)
+    public function getResource($key, $default = null)
     {
-        return $this->repository->get($key);
+        $value = $this->repository->get($key);
+
+        if (is_null($value)) {
+            return $default;
```
Comment (szymach) [inline Twig/Extension/ResourceRepository.php]: `return is_null($value) ? $default : $value;`

**Comment by JarekW** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Pull request merged**